src/hypertable.c

```diff
@@ -1368,7 +1369,7 @@ hypertable_check_associated_schema_permissions(const char *schema_name, Oid user
 }
 
 static bool
-relation_has_tuples(Relation rel)
+relation_has_tuples(const Relation rel)
```

Comment:
This expands to `struct RelationData *const`, not `const struct RelationData *`. Not sure if that is what you intended.

Reply:
Missed this one too.